ui/components/multichain/asset-picker-amount/asset-picker-amount.tsx

```diff
@@ -194,7 +194,7 @@ export const AssetPickerAmount = ({
       type: asset.type as AssetType.NFT,
       tokenId: asset.details.tokenId,
       image: asset.details.image,
-      symbol: asset.details.symbol,
+      symbol: (asset.details.symbol || asset.details.name) ?? undefined,
```

Comment:
What's the intention here?

The `??` operator only works for `null` and `undefined`, so that `?? undefined` at the end will only do anything if `symbol` is falsy and `name` is `null`, otherwise it'll already be set to `undefined`.

If you wanted to set it to `undefined` instead of `''`, then the `||` operator is necessary.

Reply:
the `?? undefined ` is mainly done to satisfy the linter because `symbol` is of type `string` or `undefined` in the jsdocs;